backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -894,6 +901,20 @@ trait StandardAsyncExecutionActor
     */
   def getTerminalEvents(runStatus: StandardAsyncRunState): Seq[ExecutionEvent] = Seq.empty
 
+  /**
+    * Get the min and max event times from a terminal run status
+    *
+    * @param runStatus The terminal run status, as defined by isTerminal.
+    * @return The min and max event times, if events exist.
+    */
+  def getStartAndEndTimes(runStatus: StandardAsyncRunState): Option[StartAndEndTimes] = None
+
+  /**
+    * The cloud platform of the job, if its running on a cloud provider.
+    *
+    */
+  def cloudPlatform: Option[Platform] = None
```

Comment:
We already have a `cromwell.backend.BackendLifecycleActorFactory#platform` – this seems redundant, or at least inconsistently named.

Reply:
I'll take a look!

Reply:
OK, the pattern I have in mind is a `PlatformSpecific` or similar trait. It defines `def platform: Platform` with no default, to force extenders to override it. Then extend it into a `GcpPlatform` trait that returns GCP.

Next the actor factories and actors that currently have `platform`/`cloudPlatform` all update to extend the platform trait they belong to. This makes it easier to keep track of all the places that have custom per-platform behavior.

This is basically fixing a style issue in code I originally wrote, so lmk if you want help.

And as a matter of style/opinion, Cromwell supports non-cloud platforms, so it shouldn't have the word "cloud" in it ([recent Slack example](https://broadinstitute.slack.com/archives/C1EH66VCM/p1718209559101789) with Slurm and LSF).

Reply:
I was able to take care of this refactor I think it worked out nicely